test/pg-cdc-resumption/mzcompose.py

```diff
@@ -60,6 +64,12 @@ def workflow_disruptions(c: Composition) -> None:
         restart_mz_during_replication,
         fix_pg_schema_while_mz_restarts,
         verify_no_snapshot_reingestion,
+        restart_mz_after_initial_snapshot,
+        restart_mz_while_cdc_changes,
+        drop_replication_slot_when_mz_is_on,
+        drop_replication_slot_when_mz_is_off,
+        # this does not work
+        # drop_replication_slot_and_change_data_when_mz_is_off
```

Comment:
We don't expect this one to succeed, do we?

Reply:
I'm not sure, it sounds ok-ish.